satellite/orders/endpoint.go

```diff
@@ -39,9 +39,9 @@ type DB interface {
 	UpdateBucketBandwidthInline(ctx context.Context, bucketID []byte, action pb.PieceAction, amount int64, intervalStart time.Time) error
 
 	// UpdateStoragenodeBandwidthAllocation updates 'allocated' bandwidth for given storage node
-	UpdateStoragenodeBandwidthAllocation(ctx context.Context, storageNode storj.NodeID, action pb.PieceAction, amount int64, intervalStart time.Time) error
+	UpdateStoragenodeBandwidthAllocation(ctx context.Context, storageNode []storj.NodeID, action pb.PieceAction, amount int64, intervalStart time.Time) error
 	// UpdateStoragenodeBandwidthSettle updates 'settled' bandwidth for given storage node
-	UpdateStoragenodeBandwidthSettle(ctx context.Context, storageNode storj.NodeID, action pb.PieceAction, amount int64, intervalStart time.Time) error
+	UpdateStoragenodeBandwidthSettle(ctx context.Context, storageNodes storj.NodeID, action pb.PieceAction, amount int64, intervalStart time.Time) error
```

Comment:
It looks like this argument rename was supposed to be done on UpdateStoragenodeBandwidthAllocation instead.